esp_delta_ota/include/esp_delta_ota.h

```diff
@@ -0,0 +1,67 @@
+/*
+ * SPDX-License-Identifier: Apache 2.0 License
+ *
+ * SPDX-FileCopyrightText: 2023 Espressif Systems (Shanghai) CO LTD
+ */
+
+#pragma once
+
+#include "esp_err.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef void *esp_delta_ota_handle_t;
+
+// Callback for reading the source data
```

Comment:
```suggestion
// Callback for reading the base binary
```
(just to keep the terminology consistent, in some cases you use "base" and in some you use "source")

Reply:
This component can also be used for others forms of data and not just .bin. Therefore I thought 'source data' would be suitable instead of 'base binary'